### PR TITLE
Add pagination to message loading

### PR DESCRIPTION
**Pull request by leirbag95**
This pull request adds pagination functionality to the message loading section of the program. The program now loads 20 messages per page, and users can navigate through all pages using the pagination component.

Changes Made:
- Added a new pagination component to handle page navigation
- Modified the message loading function to only retrieve 20 messages per page
- Added logic to the pagination component to retrieve the correct set of messages for the selected page

Pagination screenshot

### CODE REVIEW

File: src/pages/VMView.vue [hunk 1]
```diff
@@ -76,26 +95,31 @@ export default {
...
-            let storeObj = response.messages[0].content
-            programsTmp[i].storeObj = storeObj
-          }).catch(() => {
-            this.loading = false
-          })
+          if (tx & !programsTmp[i].forgotten_by) {
+            await messages.get_messages({
+              addresses: [this.account.address],
+              hashes: [tx]
+            }).then(async (response) => {
```
Comment (BjrInt) [inline src/pages/VMView.vue:116]: Please don't use `await` AND `then` constructs in the same function. The await has no effect in this specific case, this is the same for all the other "await + then" you have in this page.

File: src/pages/VMView.vue [hunk 2]
```diff
@@ -76,26 +95,31 @@ export default {
       this.loading = true
       var params = {
```
Comment (BjrInt) [inline src/pages/VMView.vue:96]: Please do not use `var` unless it has to overcome its scope (not the case here).

**Review by BjrInt — CHANGES_REQUESTED** [review]
This review covers code written on a previous iteration. The changes for this specific feature look ok though. 

Why is pagination limited to programs only? All views could benefit from it.

File: src/pages/VMView.vue (hunk 2 above)
Comment (leirbag95) [reply src/pages/VMView.vue]: I replaced `var` with `let`

File: src/pages/VMView.vue (hunk 1 above)
Comment (leirbag95) [reply src/pages/VMView.vue]: You right, I removed it!

**Review by BjrInt — APPROVED** [review]

**Pull request merged**